docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md

```diff
@@ -0,0 +1,136 @@
+
+### New deployment workflow
+Another significant change is the way Console configurations are saved and deployed: Kubernetes **configurations will be generated and committed** to the Git repository only **during the deployment process**. 
+
+Before deploying the configurations to the cluster, users can **review** which workload changes are going to be deployed by opening the review modal. This operation gives the developer a clear view of what is going to be released on the runtime environment, and it is mandatory if the user is deploying on a production environment. 
+
+Moreover, in the new workflow the user will be able to **export Kubernetes configurations** by clicking on a download button.
+:::note  
+The review and export features are still in development and will be available in future releases.
+:::
+
+:::caution
+When deploying your configurations to the cluster, keep in mind that only the changes pushed to the repository’s default branch will be considered.
+:::
+
+The new deployment workflow also unlocks the possibility to adopt a **pull-based deployment strategy**, in which a GitOps tool is set up to automatically align the cluster’s state to the latest changes pushed to the Git repository. To find out more about pull-based deployment and its advantages, visit the [GitOps deployment strategy documentation page](/development_suite/deploy/gitops-based/index.md).
+
+## Activating the feature
+
+The GitOps support feature can be activated in different ways, depending on your needs. In order to access the feature, please request the support of a Mia-Platform referent.
+
+You can activate the `ENABLE_CONF_GENERATION_ON_DEPLOY` feature toggle on a new project or Company to enable the enhanced project workflow. The feature toggle activates the new versioning system and the generation of Kubernetes configurations at the time of deployment. You can enable it for a single project or for an entire Company, which means all projects in the Company will adopt the new workflow.
+
+If you already have a Company with some projects, and you do not wish to migrate them to the new approach just yet, you can activate the `ENABLE_CREATE_PROJECT_WITH_SAVE_CONFIG_ON_DEPLOY` feature toggle on the Company. This feature toggle makes sure that all new projects in the Company will be created with the new workflow, while leaving the existing ones untouched. 
+
+:::note
+If both feature toggles are enabled, `ENABLE_CONF_GENERATION_ON_DEPLOY` will prevail.
+:::
+
+:::info
+In order to set up a new project, follow the [project creation guide](/development_suite/set-up-infrastructure/create-project.mdx). To create a new Company or to activate feature toggles, please open a Service Request.
+:::
+
+## Manual adjustments on newly created projects
+
+After creating a new project and activating the feature toggles, there are some important manual adjustments necessary to make sure the project works correctly.
```

Comment:
It depends, if you activate the first FT on the project it necessarily means you have activated it after the project creation, because otherwise on which project would it be enabled? 
If you use the first FT on the Company, all your new and past projects will use it. If you use the second FT, instead, it will be active before the new projects are created, since it refers to the Company in which they will be created.

This sentence is meant to generically recap the previous paragraphs. I don't think it matters much how you managed to activate the workflow on the project. However it can be adjusted, maybe this way it sounds better: 

```suggestion
After enabling the new workflow on one of your freshly created projects, some essential manual adjustments are necessary to make sure the project works correctly.
```